ceph/rados/serviceability_workflows.py

```diff
@@ -225,22 +225,71 @@ def remove_custom_host(self, host_node_name: str):
         Returns:
             None | raises exception in case of failure
         """
+        status_cmd = ""
         try:
+
+            def wait_osd_operation_status(status_cmd):
+                status_flag = False
+                end_time = datetime.datetime.now() + datetime.timedelta(seconds=600)
+                while end_time > datetime.datetime.now():
+                    out, err = self.cephadm.shell([status_cmd])
```

Comment:
Any reason for executing commands with `self.cephadm.shell()` and then performing operations on data?

Reply:
Please add reason for `self.cephadm.shell()` and then performing operations on data

Reply:
The information is added.